README.md

```diff
@@ -1,158 +1,270 @@
+  <img src="images/usage-step3.png" width="377" height="383" />
+  <br />
+
+  <p><strong>Step 4:</strong> After the designer window loads, launch the project by selecting either <code>Launch (windowed)</code> or <code>Launch (full-screen)</code> in <code>Tools</code> &gt; <code>Launch Project</code>.</p>
+  <img src="images/usage-step4.png" width="361" height="206" />
+  <br />
+
+  <p><strong>Step 5:</strong> After the project has finished launching, log in with your Ignition gateway’s username and password.</p>
+  <img src="images/usage-step5.png" width="418" height="239" />
+  <br />
+
+  <p><strong>Step 6:</strong> Enter the requested information, and click <code>Begin</code> to activate the demo.</p>
+  <p>For information on obtaining your Ewon Name, please refer to <em><a href="#obtaining-an-ewons-name">Appendix 6.6, Obtaining an Ewon’s Name</a>.</em></p>
+  <img src="images/usage-step6.png" width="336" height="210" />
+
+  <!-- TROUBLESHOOTING -->
+  <h2 id="troubleshooting"><span style="font-variant: small-caps;">Troubleshooting</span></h2>
+  <p><em>For additional troubleshooting information and to find troubleshooting information not found in this guide, please visit the Ewon Ignition Demo wiki, located at <a href="https://github.com/hms-networks/FlexyIgnitionDemo/wiki" class="uri">https://github.com/hms-networks/FlexyIgnitionDemo/wiki</a>.</em></p>
+
+  <!-- TROUBLESHOOTING/TRIAL EXPIRED -->
+  <h3 id="cannot-log-in-trial-expired"><strong><span style="font-variant: small-caps;">Cannot log in, trial expired</span></strong></h3>
+  <p>If you encounter this error, you’ve likely run out of time on your Ignition gateway’s demo period. To resolve, you’ll need to open your Ignition gateway’s webpage, and click <code>Reset</code> on the Trial Mode bar. If you’re not signed in to the Ignition gateway webpage, you may need to click <code>Sign In to Reset</code> instead of <code>Reset</code>.</p>
+  <img src="images/cannotlogintrialexpired.png" width="273" height="208" />
+  <br />
+
+  <!-- TROUBLESHOOTING/TAG NOT FOUND -->
+  <h3 id="error-writing-to-tag-bad_notfound"><span style="font-variant: small-caps;">Error writing to tag, Bad_NotFound</span></h3>
+  <p>If you encounter this error, there’s a problem accessing your Ewon’s tags. To resolve, you’ll need to verify that you’ve entered the correct Ewon name when entering the demo.</p>
+  <p>If you’ve entered the correct name, but still receive this error, there may be a problem with the Ewon Connector or its configuration. For information on resolving errors with the Ewon Connector, visit the Ignition Ewon Connector GitHub repository at <a href="https://github.com/hms-networks/IgnitionEwonConnector" class="uri">https://github.com/hms-networks/IgnitionEwonConnector</a> and choose the troubleshooting section.</p>
```

Comment:
directions on where the troubleshooting section is could be more clear as you just say go to the repo, not the readme.  Most people will figure out the readme is on the main landing page but some might not.

You could link to 

https://github.com/hms-networks/IgnitionEwonConnector/blob/Ignition-8-master/readme.md

Reply:
done